### DISCUSSION TITLE
Use Riallto on Linux

### DISCUSSION THREAD

**Discussion by hecmay** [AMDResearch/Riallto#25] [Q&A] [answered]
Hi, 

The document mentions that Riallto is only supported on Windows 11 + Ubuntu WSL instance. I wonder if there is any plan to support NPU and Riallto on Linux machines. If so, when will the linux support be released?

Thanks,

**Comment by STFleming** [comment]
Hi @hecmay,

This is something that we are currently testing internally and will be releasing shortly, watch this space. 

All the best,
Shane

**Comment by mariodruiz** [comment, marked as answer]
Hi @hecmay, Linux support just landed https://github.com/AMDResearch/Riallto/releases/tag/v1.1
